pkgs/applications/editors/emacs/generic.nix

```diff
@@ -74,7 +74,7 @@
 , WebKit
 
 # Boolean flags
-, nativeComp ? true
```

Comment:
If you insist on this rename, you may as well call it `withNativeCompilation` since a) many of our other boolean flags are prefixed with `with` and b) the configuration flag is `--with-native-compilation`.

Reply:
I was thinking on it, to be honest.